config/routes.rb

```diff
@@ -68,6 +68,8 @@
     resource :archive, only: %i[create destroy], controller: 'page_archives'
   end
 
+  resources :articles, path: 'articles', only: [:index], defaults: { format: 'xss' }
```

Comment:
One last thing, could wee maybe use a different name, e.g. `pages/feed`? That way it's not confusing to anyone since we don't have an actual articles resource.

Reply:
ok.